cmd/config/internal/commands/source.go

```diff
@@ -71,7 +71,7 @@ func (r *SourceRunner) runE(c *cobra.Command, args []string) error {
 
 	var inputs []kio.Reader
 	for _, a := range args {
-		inputs = append(inputs, kio.LocalPackageReader{PackagePath: a})
+		inputs = append(inputs, kio.LocalPackageReader{PackagePath: a, IncludeJSON: true})
```

Comment:
Why not just change the `MatchFilesGlob`?

Reply:
Agree. This is simpler. Done.